### PR TITLE
Support newest data transfer service in api

### PR DESCRIPTION
**Pull request by tcibinan**
Relates to #1164.

The pull request contains changes originally presented in #1164. It only introduces changes in api module. *Notice* that the pull request should be merged after #1164 because it depends on some core classes added in the original pull request.

### CODE REVIEW

File: api/src/main/java/com/epam/pipeline/exception/SystemPreferenceNotSetException.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.epam.pipeline.exception;
```
Comment (mzueva) [inline api/src/main/java/com/epam/pipeline/exception/SystemPreferenceNotSetException.java:17]: Missing license

File: api/src/main/java/com/epam/pipeline/manager/dts/DtsClient.java [hunk 2]
```diff
@@ -19,6 +19,7 @@
 import com.epam.pipeline.controller.Result;
 import com.epam.pipeline.entity.dts.DtsClusterConfiguration;
 import com.epam.pipeline.entity.dts.DtsDataStorageListing;
+import com.epam.pipeline.entity.dts.DtsDataStorageListingRequest;
```
Comment (mzueva) [inline api/src/main/java/com/epam/pipeline/manager/dts/DtsClient.java:22]: This class shall be included into PR

**Review by mzueva — CHANGES_REQUESTED** [review]

File: api/src/main/java/com/epam/pipeline/exception/SystemPreferenceNotSetException.java (hunk 1 above)
Comment (tcibinan) [reply api/src/main/java/com/epam/pipeline/exception/SystemPreferenceNotSetException.java:17]: Added.

File: api/src/main/java/com/epam/pipeline/manager/dts/DtsClient.java (hunk 2 above)
Comment (tcibinan) [reply api/src/main/java/com/epam/pipeline/manager/dts/DtsClient.java:22]: Oops! Thanks for catching this.

**Pull request merged**